newm-server/src/main/kotlin/io/projectnewm/server/statuspages/StatusPagesInstall.kt

```diff
@@ -12,17 +12,15 @@ import org.jetbrains.exposed.dao.exceptions.EntityNotFoundException
 
 fun Application.installStatusPages() {
     install(StatusPages) {
-        exception<HttpStatusException> { call, cause ->
-            call.respondStatus(cause.statusCode, cause)
-        }
-
-        exception<EntityNotFoundException> { call, cause ->
-            call.respondStatus(HttpStatusCode.NotFound, cause)
-        }
-
         exception<Throwable> { call, cause ->
-            call.respondStatus(HttpStatusCode.InternalServerError, cause)
-            cause.captureToSentry()
+            when (cause) {
+                is HttpStatusException -> call.respondStatus(cause.statusCode, cause)
+                is EntityNotFoundException -> call.respondStatus(HttpStatusCode.NotFound, cause)
+                else -> {
+                    call.respondStatus(HttpStatusCode.InternalServerError, cause)
+                    cause.captureToSentry()
+                }
+            }
         }
     }
```

Comment:
Surprised that this wasn't working as intended since it was setup the same way as the ktor docs 🤷🏽
Good catch